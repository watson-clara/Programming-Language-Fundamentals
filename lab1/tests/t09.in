debug off
stopwords on
load tracks.txt
preprocess
count tracks
count words
count characters
