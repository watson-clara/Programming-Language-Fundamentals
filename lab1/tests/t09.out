96499
230060
1342775
